test/integration/users_signup_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class UsersSignupTest < ActionDispatch::IntegrationTest
+  test "valid signup information" do
+    get signup_path
+    assert_difference 'User.count', 1 do
+      post_via_redirect users_path, params: {user: {name: "Example User",
+                                                    email: "user@example.com",
+                                                    password: "password",
+                                                    password_confirmation: "password"}}
```

Comment:
Also check your settings for spaces after hashes.  `{ this: good }` `{this: bad}`

Reply:
The previous comment around multi-line hashes applies here too (see https://github.com/airbnb/ruby#multiline-hashes)